jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/Request.java

```diff
@@ -105,6 +105,12 @@
 public class Request implements HttpServletRequest
 {
     public static final String __MULTIPART_CONFIG_ELEMENT = "org.eclipse.jetty.multipartConfig";
+    public static final String JAKARTA_SERVLET_REQUEST_PREFIX = "jakarta.servlet.request.";
+    public static final String MULTIPART_CONFIG_ELEMENT = "org.eclipse.jetty.multipartConfig";
+    public static final String SSL_CIPHER_SUITE = "jakarta.servlet.request.cipher_suite";
+    public static final String SSL_KEY_SIZE = "jakarta.servlet.request.key_size";
+    public static final String SSL_SESSION_ID = "jakarta.servlet.request.ssl_session_id";
+    public static final String X_509_CERTIFICATE = "jakarta.servlet.request.X509Certificate";
```

Comment:
Breaks backwards compatibility.

Reply:
Which bit?  I've just added constants... Oh I see the old `__` version.  I'll deprecate that one as it should never of had the `__` prefix if public.